src/lib/components/Search/index.js

```diff
@@ -96,7 +96,7 @@ class Search extends BaseElement {
         class="web-search__input-wrapper"
         role="combobox"
         aria-expanded="${this.expanded}"
-        aria-owns="web-search-popout__list"
+        aria-controls="searchInput"
```

Comment:
oh sorry, I think we need to have both the aria-owns and aria-controls attributes.

Reply:
Just to make sure: The div should have both aria-owns and aria-controls that point to the searchinput, right?

Reply:
I think aria-owns should point at web-search-popout__list and aria-controls should point at searchInput

Reply:
ah sorry, this is a bit confusing.

the id shouldn't be `searchInput` (I don't think we use camel case ids anywhere).
I guess I'd rename it to `web-search__input` and just have it match the class name.

Reply:
oki, fixed now 🙂